src/Microsoft.Framework.ApplicationHost2/project.json

```diff
@@ -3,23 +3,62 @@
     "compilationOptions": { "define": [ "TRACE" ], "warningsAsErrors": true },
     "dependencies": {
         "Microsoft.Framework.CommandLineUtils": { "version": "1.0.0-*", "type": "build" },
-        "Microsoft.Framework.Runtime.Hosting": "1.0.0-*",
+        "Microsoft.Framework.Runtime.Common": { "version": "1.0.0-*", "type": "build" },
+        "Newtonsoft.Json": "6.0.6",
+        "Microsoft.Framework.Runtime.Roslyn.Interfaces": "1.0.0-*",
```

Comment:
Hrm. The other projects don't make this a build dependency. I'm going to leave it for now, but we can catch up later and have a "build dependencies reckoning" :)